debian/ubuntu-advantage-tools.postinst

```diff
@@ -265,6 +265,26 @@ mark_reboot_cmds_as_needed() {
     add_notice "$msg_name"
 }
 
+check_wrong_fips_metapackage_on_cloud() {
```

Comment:
Minor rename the other check_* funtions return a value and are intended for use in conditionals, this intended use different, call and 
```suggestion
notify_wrong_fips_metapackage_on_cloud() {
```